cmd/kube-apiserver/app/options/validation_test.go

```diff
@@ -106,11 +113,6 @@ func TestClusterServiceIPRange(t *testing.T) {
 			expectErrors: false,
 			options:      makeOptionsWithCIDRs("10.0.0.0/16", "3000::/108"),
 		},
-		{
-			name:         "valid v6-v4 dual stack",
-			expectErrors: false,
-			options:      makeOptionsWithCIDRs("3000::/108", "10.0.0.0/16"),
-		},
```

Comment:
why was this case dropped?

Reply:
A mistake from my part, sorry. Resolved in 44c94f98b64843562323b1b9c12297be40023bdc